components/builder-api/src/server/resources/profile.rs

```diff
@@ -128,11 +147,27 @@ fn revoke_access_token(req: HttpRequest<AppState>) -> HttpResponse {
         Err(_) => return HttpResponse::new(StatusCode::UNPROCESSABLE_ENTITY),
     };
 
+    let account_id = match authorize_session(&req, None) {
+        Ok(id) => id,
+        Err(err) => return err.into(),
+    };
+
     let mut request = AccountTokenRevoke::new();
     request.set_id(token_id);
 
     match route_message::<AccountTokenRevoke, NetOk>(&req, &request) {
-        Ok(_) => HttpResponse::Ok().finish(),
+        Ok(_) => {
+            match do_get_access_tokens(&req, account_id) {
```

Comment:
same as above - get and save the old token, delete it from cache after the DB call